tests/foreman/cli/test_errata.py

```diff
@@ -238,11 +416,21 @@ def test_negative_install_without_hc_info(self):
 
         @Assert: Error message thrown.
 
-        @caseautomation: notautomated
-
         """
+        with self.assertRaises(CLIReturnCodeError):
+            HostCollection.erratum_install({
+                'organization-id': self.org['id'],
+                'errata': [self.CUSTOM_ERRATA_ID]
+            })
+        # because of the bug we cannot verify the output of the error message
+        # it seems that hammer verify errata resource before host-collection
+        # verify that the errata package is not installed
+        for virtual_machine in self.virtual_machines:
+            self.assertFalse(
+                self._is_errata_package_installed(virtual_machine))
```

Comment:
As far as I understand `Attempt to install an erratum in a hc by not specifying hc info` - it is intended scenario, but I really don't like `because of the bug we cannot verify...`, so where bug id or proper decorator. Basically, we can omit further verification as there is no chances package can be installed on VM if you don't know even in which network it is located